src/test/java/org/terasology/replayTests/WalkLongDistanceReplayTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2018 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.replayTests;
+
+import org.junit.After;
+import org.junit.Test;
+import org.terasology.ReplayTestingEnvironment;
+import org.terasology.TestUtils;
+import org.terasology.engine.GameThread;
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.logic.location.LocationComponent;
+import org.terasology.logic.players.LocalPlayer;
+import org.terasology.math.geom.Vector3f;
+import org.terasology.math.geom.Vector3i;
+import org.terasology.recording.RecordAndReplayStatus;
+import org.terasology.registry.CoreRegistry;
+import org.terasology.world.WorldProvider;
+
+import static org.junit.Assert.assertEquals;
+
+public class WalkLongDistanceReplayTest {
+    private ReplayTestingEnvironment environment = new ReplayTestingEnvironment();
+
+    private Thread replayThread = new Thread() {
+
+        @Override
+        public void run() {
+            try {
+                String replayTitle = "WalkDistanceTest";
+                environment.openReplay(replayTitle, true);
+            } catch (Exception e) {
+                throw new RuntimeException(e);
+            }
+        }
+    };
+
+    @After
+    public void closeReplay() throws Exception {
+        environment.getHost().shutdown();
+        GameThread.reset();
+        replayThread.join();
+    }
+
+    @Test
+    public void WalkLongDistance(){
```

Comment:
This isn't a good method name. It would be better to change it to "testWalkLongDistance".